docs/build_version_doc/artifacts/0.12.0.index.html

```diff
@@ -0,0 +1,1241 @@
+<!DOCTYPE html>
```

Comment:
what are these?

Reply:
Those patch the install pages to have the pip install version number. The automated route with beautiful soup is broken. The script copies each version into their respective folders. 

We'll get rid of these when we can refactor the install page.

Reply:
Also, note that I added usage instructions in the main comments.